app/src/main/java/com/codepath/michfeng/songswiper/activities/LoginActivity.java

```diff
@@ -81,4 +93,23 @@ private void authenticateSpotify() {
         AuthorizationRequest request = builder.build();
         AuthorizationClient.openLoginActivity(this, REQUEST_CODE, request);
     }
+
+    private void waitForUserInfo() {
+        UserService userService = new UserService(queue, msharedPreferences);
+        userService.get( () -> {
+            SpotifyUser user = userService.getUser();
+            editor = getSharedPreferences("SPOTIFY",0).edit();
+            editor.putString("userid",user.id);
+
+            Log.d("STARTING", "Retrieved user information");
```

Comment:
Why is the first argument using a literal string, rather than the TAG which is on line 20?